docs-kits/kits/Connector Kit/Development View/Management-API Walkthrough/1-management-api-overview.md

```diff
@@ -0,0 +1,124 @@
+---
+id: Management API Overview
+title: Management API Overview
+description: 'Connector Kit'
+sidebar_position: 1
+---
+<!--
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# Management API Overview
```

Comment:
I think this should not be part of the KIT, is a piece of technical documentation that will fit better in the tractusx-edc documentation